test/Pantry-test.js

```diff
@@ -0,0 +1,83 @@
+import { expect } from 'chai';
```

Comment:
Good test coverage. We can revisit adding more sad path testing in the future if necessary.